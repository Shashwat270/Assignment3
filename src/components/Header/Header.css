body, html {
    height: 100%;
    width: 100%;
    margin: 0px;
  }
.header {
    color: white;
    display: flex;
    flex-direction: row;
    justify-content: center;
}
li {
    list-style-type: none;
}
li h1 {
    border: 1.5px solid white;
    margin: 10px 0 0 0;
    max-width: 40px;
    border-radius: 1000px;
    font-weight: 400;
}