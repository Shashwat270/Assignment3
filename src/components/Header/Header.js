import React, { Component } from 'react';
import './Header.css';

class Header extends Component {
  render() {
    return (
      <ul className="header">
        <li>
          <h1>BS</h1>
        </li>
      </ul>
    );
  }
}
export default Header;
