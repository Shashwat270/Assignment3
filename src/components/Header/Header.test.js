import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  it('should render a header', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  const wrapper = shallow(<Header />);
  it('should have listed items in header', () => {
    expect(wrapper.find('li').getElements().length > 0).toEqual(true);
  });
});
