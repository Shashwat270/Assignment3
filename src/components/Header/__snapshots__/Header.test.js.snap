// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render a header 1`] = `
<ul
  className="header"
>
  <li>
    <h1>
      BS
    </h1>
  </li>
</ul>
`;
